tests/test_6_writers.py

```diff
@@ -69,6 +72,10 @@ def test_write_basic():
         assert_array_equal(ImmutMorphology(os.path.join(tmp_folder, "test_write.swc")).points, expected)
         assert_array_equal(ImmutMorphology(os.path.join(tmp_folder, "test_write.h5")).points, expected)
 
+        with h5py.File(h5_out) as h5_file:
+            ok_(not '/perimeters' in h5_file.keys())
```

Comment:
pep8 guideline
```suggestion
            ok_('/perimeters' not in h5_file.keys())
```

Besides, `.keys()` should not be required.
```suggestion
            ok_('/perimeters' not in h5_file)
```